tests/cases/text/cursor_move.slint

```diff
@@ -65,8 +72,8 @@ assert_eq!(instance.get_test_anchor_pos(), 0);
 slint::testing::send_keyboard_string_sequence(&instance, &LEFT_CODE.to_string());
 assert!(!instance.get_has_selection());
 
-slint::testing::set_current_keyboard_modifiers(&instance, shift_modifier);
-slint::testing::send_keyboard_string_sequence(&instance, &HOME_CODE.to_string());
+slint::testing::set_current_keyboard_modifiers(&instance, move_mod_shift_modifier);
+slint::testing::send_keyboard_string_sequence(&instance, &UP_CODE.to_string());
```

Comment:
Wasn't the goal of this code to test the "home" or begin" button?  The fact that "up" right now goes to the begining of the text is actually a bug i'd say
Or i misunderstand things

Reply:
Yes, but the old home button actually did move to the beginning of the paragraph (ignoring line wrapping) instead of the beginning of the line.
The actual home function is not testable, since `text_input_byte_offset_for_position` is not available in a testing environment.

The whole thing has nothing to do with going to the beginning or end of the document. It just does, because there i only one paragraph.